schema.sql

```diff
@@ -0,0 +1,24 @@
+INSERT INTO SchemaVersion(version_number, date_applied) VALUES ('1.0', 'Sep-20-2025');
```

Comment:
This is a great idea. I don't see the table declaration though. Maybe you created it before hand, but the migrations should have all of the declarations.